collections/sentinel-3-l1b-olci.yaml

```diff
@@ -154,222 +154,251 @@ Summaries:
   instruments:
     - olci
   eo:bands:
-    - name: B01
+    - description: Aerosol correction, improved water constituent retrieval
+      name: B01
       center_wavelength: 0.4
       full_width_half_max: 0.015
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B02
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
```

Comment:
I think it is better to use the unit symbol (°) here and not the name to be consistent with the other files where we also use "m" and not "metre".

Reply:
Makes sense. Changed.